### PR TITLE
[4.0] Screen reader text for quickicons

### PR DESCRIPTION
**Pull request by brianteeman**
Multiple fixes to the text used for screen reader only text on the quickicons when they are set to show with counts.

- typos/spelling
- consistent capitalisation
- consistent use of a full stop
- it is a count of enabled user not a count of activated users
- it is a count of locked tables not a count of locked items
- it is a count of published categories - thats not the same thing as available categories
- plugins are enabled not active

### CODE REVIEW

File: administrator/language/en-GB/en-GB.com_checkin.ini [hunk 1]
```diff
@@ -17,9 +17,9 @@ COM_CHECKIN_ITEMS_TO_CHECK_IN_DESC="Items to check-in descending"
 COM_CHECKIN_N_ITEMS_CHECKED_IN_1="Item checked in."
 COM_CHECKIN_N_ITEMS_CHECKED_IN_MORE="%s items checked in."
 COM_CHECKIN_NO_ITEMS="There are no tables with checked out items or there are no tables with checked out items that match your search."
-COM_CHECKIN_N_QUICKICON_SRONLY="Global Checkin: %s Items are locked."
-COM_CHECKIN_N_QUICKICON_SRONLY_0="Global Checkin: No Items are locked."
-COM_CHECKIN_N_QUICKICON_SRONLY_1="Global Checkin: One Item is locked."
+COM_CHECKIN_N_QUICKICON_SRONLY="Global Checkin: %s tables have locked items."
+COM_CHECKIN_N_QUICKICON_SRONLY_0="Global Checkin: No tables have locked items"
```
Comment (Quy) [inline administrator/language/en-GB/en-GB.com_checkin.ini]:
```suggestion
COM_CHECKIN_N_QUICKICON_SRONLY_0="Global Checkin: No tables have locked items."
```

**Comment by brianteeman** [comment]
retrigger drone as its still useless

**Comment by wilsonge** [comment]
Thanks!

**Comment by brianteeman** [comment]
Thx

**Pull request merged**